internal/web3ext/web3ext.go

```diff
@@ -18,6 +18,7 @@
 package web3ext
 
 var Modules = map[string]string{
+	"account":    Account_JS,
```

Comment:
`account` is very ambiguous,  lets call it `accounting`

Reply:
@JerzyLa I wouldn't call it ```account``` or ```accounting``` because of the personal RPC methods and the accounts module of web3. Maybe it would be better to call it ```bzz``` or ```swarm```.

Reply:
ok, thanks :+1:

Reply:
to be honest even `accounting` is over-generalised - let's not forget this is in the context of entire `web3`